### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by ailadson** [jhulcher/picstagram#1]
1) I'd curious if the Follows resource, as currently being used, will cause things to be overly complex? Would is be simpler to treat it like a join table and make 'followed_users' and 'following_users' associations on the User model instead. This way, you don't have to work with a follows store. If you want, let's talk more about this in person.

2) I'd do CSS incrementally instead of saving it all for the end.

3) For image storage, you can also checkout cloudinary. Regardless of which library you choose for that, get to know the documentation ASAP.

**Comment by jhulcher** [comment]
@ailadson Yes, I'd love to speak in person about way to make the project more efficient. I had intended to use the Follows table as a join. But if I don't need a store to do that, that that's great.

I also agree on the topic of CSS and plan to spend more than just 1 day on it.

I will try to take a look at cloudinary soon.

**Comment by jhulcher** [comment]
@ailadson I've updated the proposal.

1) I've changed the schema in order to use the Follows table as a Join. I removed the follows store from the proposal.

2) I plan to work on CSS throughout the project, whenever there is time or as needed. I also plan to continue working on CSS throughout the holiday break. I have updated the proposal to reflect this.

3) I have updated the schema to reflect cloudinary's data types. I included cloudinary and figaro as necessities.
